docs/CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ These will eventually replace `TelemetryIngress.URL` and `TelemetryIngress.Serve
 
 - `P2P.V1` is now disabled (`Enabled = false`) by default. It must be explicitly enabled with `true` to be used. However, it is deprecated and will be removed in the future.
 - `P2P.V2` is now enabled (`Enabled = true`) by default.
+- `L2Suggested` mode is now called `SuggestedPrice`
```

Comment:
Also I just realized the release branch for 2.7.0 has already been made so you may need to move this changelog under `[dev]` which will get picked up in the next release

Reply:
Done